### PR TITLE
added json_decode filter

### PR DESCRIPTION
**Pull request by spektra2147**

### CODE REVIEW

File: addons/default/visiosoft/advs-module/src/AdvsModulePlugin.php [hunk 1]
```diff
@@ -136,6 +136,12 @@ function ($array) {
                     return null;
                 }
             ),
+            new \Twig_SimpleFilter(
+                'json_decode',
+                function ($array) {
+                    return json_decode($array, true);
```
Comment (muammertop) [inline addons/default/visiosoft/advs-module/src/AdvsModulePlugin.php:142]: twig json_decode not present by default?

Comment (spektra2147) [reply addons/default/visiosoft/advs-module/src/AdvsModulePlugin.php:142]: there is encode. No decode @muammertop

Comment (muammertop) [reply addons/default/visiosoft/advs-module/src/AdvsModulePlugin.php:142]: i used yesterday @spektra2147

Comment (spektra2147) [reply addons/default/visiosoft/advs-module/src/AdvsModulePlugin.php:142]: ok now I'm checking again.

Comment (spektra2147) [reply addons/default/visiosoft/advs-module/src/AdvsModulePlugin.php:142]: can be used as a function.  `json_decode(test_json,true)`

**Pull request closed without merging**